src/codemagic/tools/firebase_app_distribution/errors.py

```diff
@@ -0,0 +1,5 @@
+from codemagic import cli
+
+
+class FirebaseError(cli.CliAppException):
```

Comment:
Considering recent renaming stream then this should be `FirebaseAppDistributionError`.